VectorService.md

```diff
@@ -0,0 +1,142 @@
+VectorService
+=============
+
+#### Contents
+
+- [Description](#description)
+- [Request](#request)
+- [Request Structure](#request-structure)
+- [Sample Request](#sample-request)
+- [Response Structure](#response-structure)
+- [Versioning](#versioning)
+
+## Description
+
+This service provides vector representations for documents such as a Job or Resume, and for Carotene IDs with optional skills.
+
+
+The service is located at https://api.careerbuilder.com/core/document/vectors.  This service requires CareerBuilder OAuth 2.0 client credentials. For more information please see the authorization [documentation](/Readme.md#access).
+
+Requests can be sent in the form of GET or POST (form or json).
+
+## Request
+
+A request can be broken out into two distinct types identified through a parameter called `vector_type`. Available  `vector_type`'s are: **carotene_skills** and **semantic_fingerprint**.
+
+### Carotene Skills
+
+| Parameter  | Type | Required |  Description |
+|------------|------|----------|--------------|
+| carotene_id | String | True | A carotene ID as provided by our Job Classification services which uniquely identifies a job title.
+| carotene_version   | String | True | The version of carotene the ID is associated with.
+| skills | List | False | A list of skill objects.
+
+Each skill object is made up of a unique number/letter identifier provided as a String.
+
+| Parameter  | Type | Required |  Description |
+|------------|------|----------|--------------|
+| id | String | False | A Skill ID as provided from our Skills tagging service [here](https://github.com/careerbuilder/DataScienceAPIDocumentation/blob/master/Skills.md)
+
+#### CaroteneSkills Only Request Sample
+```
+{
+  "carotene_id": "11.0",
+  "carotene_version": "CAROTENEV3",
+  "skills": [
+    {
+      "id": "KSH9TO"
+    }
```

Comment:
Now that I pay attention to the request format. Was it not possible to pass Skill IDs as simply an array of strings instead of putting objects in the array?

Reply:
hmm I think we do this purposefully because it caused us some pain in another service. @glpayson do you recall?

Reply:
@psequeirag Lamar responded to me over the weekend, and the reason why we chose to do this is to allow more flexibility in the future should more fields i.e., name will be needed for vector calculations.